make_format_args error: cannot bind non-const lvalue reference to an rvalue

/root/zedio/zedio/log/logger.hpp: In instantiation of ‘void zedio::log::detail::BaseLogger<DeriverLogger>::format(const zedio::log::detail::FmtWithSourceLocation&, Args&& ...) [with zedio::log::detail::LogLevel LEVEL = zedio::log::detail::LogLevel::Error; Args = {std::basic_string_view<char, std::char_traits<char> >}; DeriverLogger = zedio::log::detail::ConsoleLogger]’:
/root/zedio/zedio/log/logger.hpp:88:32:   required from ‘void zedio::log::detail::BaseLogger<DeriverLogger>::error(zedio::log::detail::FmtWithSourceLocation, Args&& ...) [with Args = {std::basic_string_view<char, std::char_traits<char> >}; DeriverLogger = zedio::log::detail::ConsoleLogger]’
/root/zedio/tests/tcp_socket_test.cpp:37:5:   required from here
/root/zedio/zedio/log/logger.hpp:125:62: error: cannot bind non-const lvalue reference of type ‘std::basic_string_view<char>&’ to an rvalue of type ‘std::basic_string_view<char>’
  125 |                       std::vformat(fmt, std::make_format_args(std::forward<Args>(args)...))});
      |                                         ~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/include/c++/13.2.1/format:3411:28: note:   initializing argument 1 of ‘auto std::make_format_args(_Args& ...) [with _Context = basic_format_context<__format::_Sink_iter<char>, char>; _Args = {basic_string_view<char, char_traits<char> >}]’
 3411 |     make_format_args(_Args&... __fmt_args) noexcept
      |                      ~~~~~~^~~~~~~~~~~~~~
[22/46] Building CXX object tests/CMakeFiles/queue_test.dir/queue_test.cpp.o
ninja: build stopped: subcommand failed.
make: *** [Makefile:3: build] Error 1

# g++ --version
g++ (GCC) 13.2.1 20240417

--- a/zedio/log/logger.hpp
+++ b/zedio/log/logger.hpp
@@ -95,7 +95,7 @@ public:
 
 private:
     template <LogLevel LEVEL, typename... Args>
-    void format(const FmtWithSourceLocation &fwsl, Args &&...args) {
+    void format(const FmtWithSourceLocation &fwsl, const Args &...args) {
         static thread_local std::array<char, 64> buffer{};
         static thread_local time_t               last_second{0};
 
@@ -122,7 +122,7 @@ private:
                       util::get_current_thread_name(),
                       sl.file_name(),
                       sl.line(),
-                      std::vformat(fmt, std::make_format_args(std::forward<Args>(args)...))});
+                      std::vformat(fmt, std::make_format_args(args...))});
     }
 
 private:

